xml/System.Numerics/Complex.xml

```diff
@@ -2877,11 +2988,26 @@ This API is not CLS-compliant.</summary>
         <Parameter Name="right" Type="System.Numerics.Complex" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="left">The double-precision real value to multiply.</param>
+        <param name="right">The complex value to multiply.</param>
+        <summary>Multiplies a specified double-precision real number by a specified complex number.</summary>
+        <returns>The product of <paramref name="left" /> and <paramref name="right" />, as a complex number.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The multiplication of a real number (which can be regarded as the complex number a + 0i) and a complex number (c + di) takes the following form:
+
+ac + adi
+
+Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Multiply%28System.Double%2CSystem.Numerics.Complex%29?displayProperty=nameWithType> equivalent method instead.
+
+There are implicit casts from double -> Complex, so it's currently possible to write e.g. 2.5 * z for Complex z, but in addition to the cast, this costs 6 flops for a Complex * Complex multiply, while it's possible to implement a double * Complex multiply directly in 2 flops.
```

Comment:
Does this paragraph should be moved to `MemerGroup` or should be also stated in `op_Multiply(Complex, double)` overload?

Reply:
Oh nice catch. I wasn't intending to keep that comment. I forgot to delete it.

Reply:
```suggestion
```